code/src/components/Expectations.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+const Expectations = ({ expectations, setExpectations, nextSlide, backSlide }) => {
+  const feedbackValues = ['superduper', 'great', 'standard', 'horrible']
```

Comment:
Glad to see that you added a const with array here. An alternative way is to set it up before the function itself (between the import and 'const Expectations).  Good work!